src/app/views.py

```diff
@@ -1290,13 +1290,37 @@ def archiveRequests(request, license_id=None):
     """ View for archive license requests
     returns archive_requests.html template
     """
+    context_dict = {}
     if request.method == "POST" and request.is_ajax():
-        archive = request.POST.get('archive', False)
-        license_id = request.POST.get('license_id', False)
-        if license_id:
-            LicenseRequest.objects.filter(pk=license_id).update(archive=archive)
+        if request.user.is_authenticated():       
+            user = request.user
+            """ Getting user info for submitting github issue """
+            github_login = user.social_auth.get(provider='github')
+            token = github_login.extra_data["access_token"]
+            username = github_login.extra_data["login"]
+            test = requests.get('https://api.github.com/repos/spdx/tools-python/collaborators/{username}' , headers={'Authorization': 'token {}'.format(token) })
+            if ((test.status_code == 200) or (test.status_code == 204)):
```

Comment:
Add try except here, since this api may result in error. Also, if status code returned is not 200/204, then also we should show some msg to the user.

Reply:
@rtgdk What do you think should be the changes in submit license. In the current version, the  user not logged in gets a message as to he must login in before submitting. If I hide the submit button from the user not logged in, He will not be able to get an idea as to what to do to submit the license.

Reply:
In the backend the user gets a message that permission is denied. but no message should be displayed in the frontend as there is no option(button) for archive.

Reply:
Let's leave submit license page then.